### PR TITLE
Improve `index.php` and `setup/index.php`

### PR DESCRIPTION
**Pull request by MauricioFauth**
Simplifies the `index.php` and `setup/index.php` entry points with some refactoring to the `Common` class.

### CODE REVIEW

File: setup/index.php [hunk 1]
```diff
@@ -3,41 +3,32 @@
...
+    );
 }
 
-Core::fatalError(sprintf(
-    __('Error 404! The page %s was not found.'),
-    '[code]' . htmlspecialchars($route) . '[/code]'
-));
+require AUTOLOAD_FILE;
+
+chdir(ROOT_PATH);
```
Comment (williamdes) [inline setup/index.php:32]: I remember working on #16187 
Is changing dir still needed ?

**Review by williamdes — APPROVED** [review]
Neat !

File: setup/index.php (hunk 1 above)
Comment (MauricioFauth) [reply setup/index.php]: You're right. This is not needed anymore.

**Review by williamdes — APPROVED** [review]
🚢

**Pull request merged**